README.adoc

```diff
@@ -5,65 +5,70 @@
 :maven-documentation-profile-link: http://maven.apache.org/guides/introduction/introduction-to-profiles.html
 :hivemq-support: http://www.hivemq.com/support/
 :hivemq-listener: https://www.hivemq.com/docs/hivemq/4.4/user-guide/listeners.html#tcp-listener
+:hivemq-extension-download: https://www.hivemq.com/extension/file-rbac-extension/
```

Comment:
you can have the GitHub link here

Reply:
I aligned the links on all extensions to be the marketplace.